package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "sexual-health-service-data-combiner",
+  "version": "0.1.0",
+  "description": "Retrieves multiple JSON files, combines them into a single file and uploads it into Azure Blob storage",
+  "main": "app.js",
+  "scripts": {
+    "coverage-check": "istanbul check-coverage --config .istanbul.yml",
+    "coverage-check-unit": "istanbul check-coverage --config .istanbul.unit.yml",
+    "coverage-generate-unit": "yarn istanbul cover -- _mocha -- --recursive test/unit",
+    "coverage-generate": "yarn istanbul cover -- _mocha -- --recursive",
+    "coverage-upload-coveralls": "cat ./coverage/lcov.info | coveralls",
+    "git-hook": "yarn lint && yarn coverage-generate-unit && yarn coverage-check-unit",
+    "istanbul": "NODE_ENV=test istanbul",
+    "lint": "eslint --ext .js,.json .",
+    "postrewrite": "yarn git-hook",
+    "precommit": "yarn git-hook",
+    "prepush": "yarn git-hook && yarn snyk test",
+    "start": "node app.js",
+    "start-watch": "nodemon app.js | ./node_modules/bunyan/bin/bunyan & nodemon -e scss -x yarn build-css -- compact",
+    "test": "NODE_ENV=test mocha --recursive test",
+    "test-ci": "yarn lint && yarn coverage-generate && yarn coverage-check && yarn coverage-upload-coveralls",
+    "test-integration": "NODE_ENV=test mocha --recursive test/integration",
+    "test-unit": "NODE_ENV=test mocha --recursive test/unit",
+    "test-watch": "yarn test --watch --reporter min"
+  },
+  "keywords": [
+    "azure",
+    "json",
+    "nhsuk"
+  ],
+  "author": "Steve Hunt",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "coveralls": "^3.0.0",
+    "eslint": "^4.6.1",
+    "eslint-config-airbnb-base": "^12.0.0",
+    "eslint-config-nhsuk": "^0.13.0",
+    "eslint-plugin-import": "^2.6.1",
+    "eslint-plugin-json": "^1.2.0",
+    "eslint-plugin-mocha": "^4.11.0",
+    "eslint-watch": "^3.1.2",
+    "husky": "^0.14.2",
+    "istanbul": "^1.1.0-alpha.1",
+    "mocha": "^5.0.0",
+    "nock": "^9.0.14",
+    "snyk": "^1.40.2"
+  },
+  "dependencies": {
+    "azure-storage": "^2.4.0",
+    "moment": "^2.18.1",
+    "nhsuk-bunyan-logger": "^1.4.0",
+    "node-schedule": "^1.2.4",
+    "pretty-hrtime": "^1.0.3",
+    "request-promise-native": "^1.0.5",
```

Comment:
there's a warning on install of warning "request-promise-native@1.0.5" has unmet peer dependency "request@^2.34". I think you also need to explicitly add request too.

Reply:
Yeah I think that is more a problem with the configuration of the package in question. https://github.com/request/request-promise-native/blob/master/package.json#L41 specifies `request@2.34` is required but I'd hazard a guess that it should be more lenient than that and probably be `^2.34.0` or something similar to signify any version 2.
More background on [peer dependencies](https://nodejs.org/en/blog/npm/peer-dependencies/).
Given request is installed, the package works and there are plenty of [issues](https://github.com/yarnpkg/yarn/issues/5347#issuecomment-376219812) about yarn not installing peer dependencies I'd be inclined to not do anything. Unless that isn't acceptable?